ConsoleApp2/ConsoleApp2/Program.cs

```diff
@@ -0,0 +1,233 @@
+﻿using System;
+
+namespace Game
+{
+    public static class Exit
+    {
+        public static bool gameLive=false;
+    }
+    class Program
+    {
+        public bool gameLive = false;
+        public class Point
+        {
+            public int X { get; set; }
+            public int Y { get; set; }
+            public Point(int x, int y)
+            {
+                this.X = x;
+                this.Y = y;
+            }
+            public static Point Addition(Point point1, Point point2)
+            {
+                Point point3 = new Point(0, 0);
+                point3.X = point1.X + point2.X;
+                point3.Y = point1.Y + point2.Y;
+                return point3;
+            }
+        }
+
+        public class Snake : List<Point>
+        {
+            public Point direction { get; set; }
+            public Point track { get; set; }
+
+            public Snake()
+            {
+                this.Add(new Point(1, 1));
+                this.Add(new Point(2, 1));
+                this.Add(new Point(3, 1));
+                direction = new Point(1, 1);
+            }
+            public void move()
+            {
+                track = this.Last();
+
+                this.RemoveAt(this.Count - 1);
+                this.Insert(0, Point.Addition(this.ElementAt(0), direction));
+
+                try
+                {
+                    Console.SetCursorPosition(track.X, track.Y);
```

Comment:
set cursor position throws several exceptions, every exception will lead to game over, it's not ok
![image](https://user-images.githubusercontent.com/14860319/148644389-74f2d79e-2624-440f-a441-c86b164b1646.png)